virtualizarr/readers/tiff.py

```diff
@@ -1,20 +1,48 @@
-import warnings
-from pathlib import Path
-from typing import Iterable, Mapping, Optional
+from __future__ import annotations
 
-from xarray import Dataset, Index
+import dataclasses
+import math
+from typing import TYPE_CHECKING, Iterable, Mapping, Optional
+
+import numcodecs.registry as registry
+from async_tiff import TIFF
+
+from virtualizarr.codecs import numcodec_config_to_configurable
+from virtualizarr.manifests import ChunkManifest, ManifestArray
+from virtualizarr.manifests.utils import create_v3_array_metadata
+
+if TYPE_CHECKING:
+    from async_tiff._ifd import ImageFileDirectory
```

Comment:
I think this should be exported from the top level now?
```suggestion
    from async_tiff import ImageFileDirectory
```